mylyn.tasks/connectors/gitlab/org.eclipse.mylyn.gitlab.ui/src/org/eclipse/mylyn/gitlab/ui/GitlabImages.java

```diff
@@ -0,0 +1,85 @@
+package org.eclipse.mylyn.gitlab.ui;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+import org.eclipse.core.runtime.Platform;
+import org.eclipse.jface.resource.ImageDescriptor;
+import org.eclipse.jface.resource.ImageRegistry;
+import org.eclipse.mylyn.commons.ui.CommonImages;
+import org.eclipse.mylyn.internal.commons.ui.CommonsUiConstants;
+import org.eclipse.swt.graphics.Image;
+import org.osgi.framework.Bundle;
+
+public class GitlabImages {
```

Comment:
This class looks strange for me. It tries to use `ImageRegistry` and at the same time duplicates its functionality. Consider replacement with `ImageRegistry`

Reply:
Duplicated functionality is removed with next commit.